hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3287,4 +3287,13 @@
       will create intermediate directories.
     </description>
   </property>
+  <property>
+    <name>ozone.om.snapshot.cache.max.size</name>
+    <value>200</value>
```

Comment:
200 could be a lot of open rocksDB instances. I guess a cache of last 10-20 accessed snapshots should be enough. If anyone is making excessive use of concurrently open/actively read snapshots, they can tune it.

Reply:
thank you for tuning it down to 10